addons/website/static/src/systray_items/mobile_preview.xml

```diff
@@ -1,9 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 <t t-name="website.MobilePreviewSystray">
-    <div class="o_mobile_preview o_menu_systray_item d-none d-md-block"
-         t-att-class="{ 'o_mobile_preview_active': this.state.isMobile }">
-        <a href="#" accesskey="v" class="o_nav_entry"
+    <div class="o_mobile_preview o_menu_systray_item d-none d-md-flex">
+        <a href="#" accesskey="v" class="o_nav_entry mx-1 px-3 is_full_height" t-att-class="{ 'o_mobile_preview_active': this.state.isMobile }"
```

Comment:
why does `o_mobile_preview_active` need to be moved from the `div.o_mobile_preview` to its inner `a`?

Reply:
My mistake, I’ve put it back 👍 .